token/src/state.rs

```diff
@@ -549,6 +553,7 @@ pub trait IsInitialized {
 }
 
 // Pull in syscall stubs when building for non-BPF targets
+#[cfg(feature = "program")]
```

Comment:
If these don't exist in the non-bpf, non-program configuration there will be unresolved symbols, hence the stubs